index.html

```diff
@@ -1,25 +1,32 @@
 <!doctype html>
 <html>
   <head>
-    <link rel='stylesheet' href='styles.css'>
-  </head>
+      <link rel='stylesheet' href='styles.css'>
+
+      <title>Checkpoint 01</title>
   <body>
     <header>
-      <h1>The Header</h1>
+      <h1>The Header </h1>
     </header>
+
     <aside>
       <h2>A Side Bar</h2>
     </aside>
-    <div class='main'>
+
+    <main>
       <h2>The Main Content</h2>
+
       <div class='left'>
+
 	This is the left side
-	<img src='https://jesse.sh/img/me.jpg'>
+	<img src="https://jesse.sh/img/me.jpg" alt "guy playing a guitar">
```

Comment:
You're missing an equals sign after `alt`. This might cause the page not to load properly.